tests/Env/Config/RendererTest.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+/*
+ * This file is part of the Manala package.
+ *
+ * (c) Manala <contact@manala.io>
+ *
+ * For the full copyright and license information, please refer to the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Manala\Tests\Config;
+
+use Manala\Env\Config\Config;
+use Manala\Env\Config\Renderer;
+use Manala\Env\Config\Variable\Variable;
+
+class RendererTest extends \PHPUnit_Framework_TestCase
+{
+    private static $tempdir;
+
+    public static function setUpBeforeClass()
+    {
+        self::$tempdir = sys_get_temp_dir().'/ManalaRendererTest';
+        @mkdir(self::$tempdir);
+    }
+
+    public function testRender()
+    {
+        file_put_contents(self::$tempdir.'/template', 'foo {{ placeholder }} baz');
+        $var = $this->prophesize(Variable::class);
+        $var->getReplaces()->willReturn(['{{ placeholder }}' => 'bar']);
+        $config = $this->prophesize(Config::class);
+        $config->getVars()->willReturn([$var->reveal()]);
+        $config->getTemplate()->willReturn(new \SplFileInfo(self::$tempdir.'/template'));
+
+        $this->assertSame('foo bar baz', Renderer::render($config->reveal()));
+    }
+
+    public function testRenderYamlWithReplacesAtDeep()
```

Comment:
Oops.. forget to finish it. It is for testing that dependencies are properly filled under the `app_options` level of the `group_vars/app.yml` template